### ISSUE TITLE
Windows CMD popup with NX running in the background

### ISSUE THREAD

**Issue by gotenxds** [nrwl/nx#27798]
### Current Behavior

I've seen that this issue showed up it's head multiple times in the last 2-3 years, as far as I've seen they have all been closed, well, I still encounter it, with a fresh nx install for fastify generator and serve.

This time, compared to previous tickets I've seen on this issue, the windows just keep opening randomly(?), even if I dont use the IDE actively, so it is enough that there is an active nx process somewhere.

I also made sure to check if disabling nx console helps, it does not, this happens regardless of the console.

It just keeps opening them forever a few times a minute, after a few minutes this is what I have: 
![image](https://github.com/user-attachments/assets/3447f894-36b3-4d9e-8213-299a00527d10)

I've also noticed that I now have a lot of open node process in the background that never close and I need to terminate them manually - all related esbuild (?)

![image](https://github.com/user-attachments/assets/dd85d180-b31e-40d3-b875-a04cda4e237e)

### Expected Behavior

Not to happen

### Steps to Reproduce

This is a fresh nx workspace and a fresh app, all I created is a new app using `nx generate @nx/node:application` picking fastify as an option
I also have 1 lib but I dont think that has anything to do with it.

### Nx Report

```shell
Node           : 20.17.0
OS             : win32-x64
Native Target  : x86_64-windows
npm            : 10.8.2

nx (global)        : 19.6.4
nx                 : 19.6.4
@nx/js             : 19.6.4
@nx/jest           : 19.6.4
@nx/linter         : 19.6.4
@nx/eslint         : 19.6.4
@nx/workspace      : 19.6.4
@nx/devkit         : 19.6.4
@nx/esbuild        : 19.6.4
@nx/eslint-plugin  : 19.6.4
@nx/node           : 19.6.4
@nx/react          : 19.6.4
@nrwl/tao          : 19.6.4
@nx/vite           : 19.6.4
@nx/web            : 19.6.4
typescript         : 5.5.2
---------------------------------------
Registered Plugins:
@nx/vite/plugin
@nx/eslint/plugin
@nx/jest/plugin
```

### Operating System

- [ ] macOS
- [ ] Linux
- [X] Windows
- [ ] Other (Please specify)

**Comment by clonardo** [comment]
I am also seeing this on Nx 19.7.3, and have been seeing it for some time (at least the last few months of working with 19.x, across several projects). Similar to OP, I also saw a number of old issues on this, with many auto-closed due to inactivity after a period of radio silence (not sure the auto-close thing is a good idea as longstanding issues like this fall off the list without being actioned, but I digress).


My setup:
```
Node           : 20.17.0
OS             : win32-x64
Native Target  : x86_64-windows
pnpm           : 9.9.0

nx                 : 19.7.3
@nx/js             : 19.7.3
@nx/jest           : 19.7.3
@nx/linter         : 19.7.3
@nx/eslint         : 19.7.3
@nx/workspace      : 19.7.3
@nx/cypress        : 19.7.3
@nx/devkit         : 19.7.3
@nx/eslint-plugin  : 19.7.3
@nx/nest           : 19.7.3
@nx/node           : 19.7.3
@nx/playwright     : 19.7.3
@nx/react          : 19.7.3
@nx/storybook      : 19.7.3
@nrwl/tao          : 19.7.3
@nx/vite           : 19.7.3
@nx/web            : 19.7.3
@nx/webpack        : 19.7.3
typescript         : 5.5.4
---------------------------------------
Registered Plugins:
@nx/vite/plugin
@nx/eslint/plugin
@nx/playwright/plugin
@nx/jest/plugin
@nx/webpack/plugin
```

**Comment by MaxKless** [comment]
Hey! I just pushed a fix to `19.8.1` that fixes over a hundred instances where this could've occurred. Can you update and see if the issue persist? 

Please let me know if this still happens and we can reopen the issue and I'll have a look :)
